satellite/gracefulexit/endpoint.go

```diff
@@ -252,7 +261,17 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 			if err != nil {
 				return Error.Wrap(err)
 			}
-
+			deleteMsg := &pb.SatelliteMessage{
+				Message: &pb.SatelliteMessage_DeletePiece{
+					DeletePiece: &pb.DeletePiece{
```

Comment:
Do we need to send the satellite signature with the delete message?

Reply:
The exiting node connects to the satellite and receives the delete message. There is no way (that I know of) for another storage node to connect to a node and send the delete message.